### PR TITLE
Use ios/android clients as dependency

### PR DESCRIPTION
**Pull request by mironiasty**
## Description

Update native code configuration to use dependencies from JitPack/Cocoapods with latest native libraries

## Motivation and Context

We are no longer maintain previously used `membrane-webrtc-android`/`membrane-webrtc-ios`. So we should depend on new `ios-client-sdk` and `android-client-sdk`

### CODE REVIEW

File: ios/FishjamRNClient.podspec [hunk 1]
```diff
@@ -3,7 +3,7 @@ require 'json'
 package = JSON.parse(File.read(File.join(__dir__, '..', 'package.json')))
 
 Pod::Spec.new do |s|
-  s.name           = 'MembraneWebRTC'
+  s.name           = 'FishjamRNClient'
```
Comment (balins) [inline ios/FishjamRNClient.podspec:6]: Shouldn't it be just `FishjamClient` like everywhere else?

Comment (balins) [reply ios/FishjamRNClient.podspec:6]: Also, line `15` – could you update the `git` link to https://github.com/fishjam-dev/react-native-client-sdk? As for now it's some random inactive repo

File: package.json [hunk 2]
```diff
@@ -27,7 +27,8 @@
     "react-native",
     "expo",
     "@jellyfish-dev/react-native-client-sdk",
-    "MembraneWebRTC"
+    "MembraneWebRTC",
+    "Fishjam"
   ],
   "repository": "https://github.com/membraneframework/react-native-client-sdk",
```
Comment (balins) [inline package.json:33]: Similarly, could you please update the links to https://github.com/fishjam-dev/react-native-client-sdk?

Comment (balins) [reply package.json:33]: Also, the package name is still in `jellyfish` namespace, as well as the keywords point to it (line 29). If this is intended to be fixed in another PR you can ignore it.

**Review by balins — APPROVED** [review]
LGTM, but before releasing it, we need to update obsolete `Jellyfish` references in this or another PR

File: ios/FishjamRNClient.podspec (hunk 1 above)
Comment (karkakol) [reply ios/FishjamRNClient.podspec:6]: Agreed. s.source should be updated.

Comment (karkakol) [inline ios/FishjamRNClient.podspec:6]: packages on npm uses following naming conventions:
`RNPackageName` or `react-native-package-name`

Comment (karkakol) [reply ios/FishjamRNClient.podspec:6]: so i would change it to RNFishjamClient

File: android/build.gradle
Comment (karkakol) [inline android/build.gradle:1]: I am almost 100% sure that this PR will not compile and work properly with new android-client-sdk with changed namespaces, and i see that from android files only build.gradle changed. What about other files?

Comment (karkakol) [inline android/build.gradle:1]: Namespaces and groups in this file should be updated to `fishjam` not leaved as `membrane`, eg line 5, 54

**Review by karkakol — CHANGES_REQUESTED** [review]

File: ios/FishjamRNClient.podspec (hunk 1 above)
Comment (mironiasty) [reply ios/FishjamRNClient.podspec:6]: `FishjamClient` is 'reserved' by `ios-client-sdk` pod name. So this name has to be different

File: android/build.gradle
Comment (mironiasty) [reply android/build.gradle:1]: I've tested it and it works fine. rn-client-sdk uses only membrane namespaces (as for now), so there is no surprise it still works after refactor.

Comment (mironiasty) [reply android/build.gradle:1]: This is not in scope of this PR.

**Comment by mironiasty** [comment]
> we need to update obsolete Jellyfish references in this or another PR 

Yes, but let's do it in another PR. This PR meant to only update dependencies. Not namespaces.

File: ios/FishjamRNClient.podspec
Comment (karkakol) [inline ios/FishjamRNClient.podspec:1]: Change file name.

File: package.json [hunk 3]
```diff
@@ -26,16 +26,17 @@
...
     "expo",
-    "@jellyfish-dev/react-native-client-sdk",
-    "MembraneWebRTC"
+    "@fishjam-dev/react-native-client-sdk",
+    "MembraneWebRTC",
+    "Fishjam"
   ],
-  "repository": "https://github.com/membraneframework/react-native-client-sdk",
+  "repository": "https://github.com/fishjam-dev/react-native-client-sdk",
   "author": "Angelika Serwa <angelika.serwa@swmansion.com> (https://github.com/graszka22)",
```
Comment (karkakol) [inline package.json:34]: Know that it is not your code but lets update author to software mansion.

**Review by karkakol — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (mironiasty) [reply package.json]: I was planning to do that before releasing new version of lib. But updated it here

File: package.json (hunk 2 above)
Comment (mironiasty) [reply package.json]: Yes, all Jellyfish references will updated in separate PR (it requires a little more work)

**Review by karkakol — APPROVED** [review]

**Pull request merged**